bindings/c/include/opendal.h

```diff
@@ -628,8 +628,8 @@ struct opendal_result_read opendal_operator_blocking_read(const struct opendal_o
  *
  * * If the `path` points to NULL, this function panics, i.e. exits with information
  */
-struct opendal_result_reader opendal_operator_blocking_reader(const struct opendal_operator_ptr *ptr,
-                                                              const char *path);
+struct opendal_result_reader opendal_operator_reader(const struct opendal_operator_ptr *ptr,
```

Comment:
I'm not sure do we need to naming it as `opendal_result_operator_reader`? To match the same style like `opendal_result_reader_read`.

Reply:
I agree with you